website/docs/r/policy.html.markdown

```diff
@@ -11,7 +11,7 @@ description: |-
 Lacework provides a highly scalable platform for creating, customizing, and managing custom policies
 against any datasource that is exposed via the Lacework Query Language (LQL).
 
-For more information, see the [Policy Overview Documentation](https://docs.lacework.com/custom-policies-overview).
+For more information, see the [Policy Overview Documentation](https://docs.lacework.net/console/custom-policy-overview).
```

Comment:
nice! Thank you
<img src="https://media3.giphy.com/media/ZfK4cXKJTTay1Ava29/giphy.gif"/>